docs/topics/ksp/ksp-why-ksp.md

```diff
@@ -58,7 +58,7 @@ is fairly quick. Almost all of the build overhead resides in the stub generation
 reduce the time spent in the compiler by 25%.
 
 For performance evaluation, we implemented a [simplified version](https://github.com/google/ksp/releases/download/1.4.10-dev-experimental-20200924/miniGlide.zip)
-of [Glide](https://github.com/bumptech/glide) in KSP to make it generate code for the [Tachiyomi](https://github.com/inorichi/tachiyomi) project.
+of [Glide](https://github.com/bumptech/glide) in KSP to make it generate code for the Tachiyomi project.
```

Comment:
I haven't found an actual link.

Reply:
I found this one: https://github.com/tachiyomiorg